Gemfile.lock

```diff
@@ -394,9 +394,9 @@ GEM
       execjs (>= 0.3.0, < 3)
     unf (0.1.4)
       unf_ext
-    unf_ext (0.0.8)
+    unf_ext (0.0.8.1)
     unicode-display_width (2.1.0)
-    view_component (2.49.1)
+    view_component (2.50.0)
```

Comment:
Hold off on merging until we have a new blacklight version that's compatibility with this.